runtime/moonbeam/src/lib.rs

```diff
@@ -372,6 +380,7 @@ impl pallet_transaction_payment::Config for Runtime {
 	type WeightToFee = ConstantMultiplier<Balance, ConstU128<{ currency::WEIGHT_FEE }>>;
 	type LengthToFee = LengthToFee;
 	type FeeMultiplierUpdate = SlowAdjustingFeeUpdate<Runtime>;
+	type WeightInfo = ();
```

Comment:
We need proper weights here.

Reply:
Reverted. Working on a proper solution involving a new cherry-pick on `polkadot-sdk`.

Reply:
Addressed in:
- moonbeam: https://github.com/moonbeam-foundation/moonbeam/pull/3224/commits/ea3b172e3c387a90d4b1df47865e3ec040f12415
- polkadot-sdk: https://github.com/moonbeam-foundation/polkadot-sdk/commit/a1e8632189c14583581445482a2f6002e85385d9

Reply:
Need to add the weight info.